### ISSUE TITLE
[Nano] load pytorch_fx model will get an error when using 2.2.0b20221212 version

### ISSUE THREAD

**Issue by NingWang5** [intel-analytics/BigDL#6968]
Below is my code to load ckpt of pytorch_fx:
``` 
config = {'input_feature_num':8,
          'output_feature_num':1,
          'past_seq_len':48,
          'future_seq_len':1,
          'kernel_size':3,
          'repo_initialization':True,
          'dropout':0.1,
          'seed': 0,
          'num_channels':[16]*3
          }
model = model_creator(config)
loss = torch.nn.MSELoss()
optimizer = torch.optim.Adam(lr=0.001, params=model.parameters())

lit_model = Trainer.compile(model, loss, optimizer)

fx_q_model = InferenceOptimizer.load("./gpu_ckpt/fx_q_model", lit_model)
```

It will successed when using 2.2.0b20221128 version, but get an error when using the newest version (2.2.0b20221212), and the error log is as following:

![image](https://user-images.githubusercontent.com/113402880/207269515-c88419fc-163a-4a38-b839-e33df42a2434.png)

**Comment by rnwang04** [comment]
Do you save your model again with the newest version (2.2.0b20221212)? or you just load the model which is saved in 2.2.0b20221128 version.

**Comment by NingWang5** [comment]
THX，nano versions of save and load are not unified, no issue now.

**Comment by TheaperDeng** [comment]
> THX，nano versions of save and load are not unified, no issue now.

well that's a good catch for us. Although we may not support some nightly built version's checkpoint, but should we support 2.1.0's checkpoint in 2.2.0? (no strong idea about this) @rnwang04

**Comment by rnwang04** [comment]
> > THX，nano versions of save and load are not unified, no issue now.
> 
> well that's a good catch for us. Although we may not support some old nightly built version's checkpoint, but should we support 2.1.0's checkpoint in 2.2.0? (no strong idea about this) @rnwang04

I agree, maybe we should support such behavior of save & load and I think it's not too difficult for us.
